preprocess.py

```diff
@@ -75,15 +74,15 @@ def norm_data(args):
 
 def run_preprocess(args):
 	input_folders = norm_data(args)
-	output_folder = os.path.join(args.base_dir, args.output)
+	output_folder = os.path.join(os.getcwd(), args.output)
```

Comment:
I'd suggest to start using `from pathlib import Path` and then `output_folder = Path.cwd() / args.output`.
Worth considering perhaps..